riot-headers.h

```diff
@@ -165,11 +168,11 @@
 
 /* packages */
 #ifdef MODULE_NIMBLE_HOST
-#  include "host/ble_gatt.h"
-#  ifndef IS_C2RUST
-#    include "host/ble_gap.h"
-#  endif
-#  include "host/ble_hs_adv.h"
+#include "host/ble_gatt.h"
+#ifndef IS_C2RUST
+#include "host/ble_gap.h"
+#endif
+#include "host/ble_hs_adv.h"
```

Comment:
Please don't alter unrelated code needlessly.

Reply:
fixed, sry did not realize that the auto formatter had kicked in.